docs/rpc_package.md

```diff
@@ -18,7 +18,7 @@ Methods in the RPC package map one-to-one to the corresponding Tezos RPC API end
 
 import { RpcClient } from '@taquito/rpc';
 
-const client = new RpcClient(' https://ithacanet.ecadinfra.com/', 'NetXjD3HPJJjmcd');
+const client = new RpcClient(' https://jakartanet.ecadinfra.com/', 'NetXjD3HPJJjmcd');
```

Comment:
wrong chain_id, should be NetXLH1uAxK7CCh

Reply:
fixed